cmd/agent/install_script.sh

```diff
@@ -363,15 +363,20 @@ elif [ "$OS" = "SUSE" ]; then
     fi
   fi
 
-  gpgkeys=''
-  separator='\n       '
-  for key_path in "${RPM_GPG_KEYS[@]}"; do
-    gpgkeys="${gpgkeys:+"${gpgkeys}${separator}"}https://${yum_url}/${key_path}"
-  done
-  if [ "$agent_major_version" -eq 6 ]; then
-    for key_path in "${RPM_GPG_KEYS_A6[@]}"; do
+  SUSE_VER=$( (cat /etc/SuSE-release 2>/dev/null; cat /etc/SUSE-brand 2>/dev/null) | grep VERSION | tr . = | cut -d = -f 2 | xargs echo)
```

Comment:
Why do you need the `xargs echo` in the end? Isn't that redundant?

Reply:
Unfortunately it's not. It trims whitespace around the input. Maybe `xargs echo` would be more explicit? `echo` is the default, though.

Reply:
Ah, I see. We could add a comment about this if you think it won't be obvious to other people too